qiskit/transpiler/transpile_config.py

```diff
@@ -30,6 +30,9 @@ class TranspileConfig(BaseModel):
             optimization level. 0 means no transformation on the circuit. Higher
             levels may produce more optimized circuits, but may take longer.
     """
-    def __init__(self, optimization_level, **kwargs):
+    def __init__(self, optimization_level=None, **kwargs):
```

Comment:
and let's expand kwargs to be args with default set to None. So that at least the structure of this class is clear and not just anything can go inside it.